src/components/ResourceInstance/ResourceInstanceDetails/ResourceInstanceDialog.jsx

```
@@ -76,6 +85,18 @@ export default function ResourceInstanceDialog(props) {
             <FieldLabel sx={{ mt: "5px", fontSize: "16px" }}>
               {subtitle}
             </FieldLabel>
+            <FieldDescription
+              dangerouslySetInnerHTML={{ __html: message }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)